src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -193,10 +172,12 @@ void DisplayWindowMupen64plus::_readScreen2(void * _dest, int * _width, int * _h
 	if (_dest == nullptr)
 		return;
 
+	m_glInfo.init();
```

Comment:
It's not a global or a single ton. It has to be re-created here. It would had been a pain to use the same instance of GLInfo in both places.